Ds3Examples/Ds3ExampleClient.cs

```diff
@@ -0,0 +1,503 @@
+            }
+            return true;
+        }
+
+
+        public IEnumerable<DS3GetObjectsInfo> GetObjects(string bucketName, string objectName, string ds3ObjectId, long length, long offset, DS3ObjectTypes type, long version)
+        {
+            var request = new Ds3.Calls.GetObjectsRequest()
+            {
+                BucketId = bucketName,
+                ObjectName = objectName,
+                ObjectId = ds3ObjectId,
+                Length = length,
+                Offset = offset,
+                ObjectType = type,
+                Version = version
+            };
+            var response = _client.GetObjects(request);
+            foreach (var ds3Object in response.Objects)
+            {
+                yield return ds3Object;
+            }
+        }
+  
+        public GetSystemInformationResponse GetSystemInfo()
+        {
+            var request = new Ds3.Calls.GetSystemInformationRequest();
+            return _client.GetSystemInformation(request);
+        }
+
+        public VerifySystemHealthResponse VerifySystemHealth()
+        {
+            var request = new Ds3.Calls.VerifySystemHealthRequest();
+            return _client.VerifySystemHealth(request);
+        }
+
+
+        #region main()
+
+        private static TraceSwitch clientSwitch = new TraceSwitch("clientSwitch","test switch");
+
+        static void Main(string[] args)
+        {
+            if (clientSwitch.TraceInfo) { Trace.WriteLine("Starting Ds3ExampleClient main()"); }
+            if (clientSwitch.TraceWarning) { Trace.WriteLine("Almost out of coffee"); }
+            if (clientSwitch.TraceVerbose) { Trace.WriteLine("I think I should share some of my feelings..."); }
+
+            // set the following environment variables or pass in from App.config
+            string endpoint = Environment.GetEnvironmentVariable("DS3_ENDPOINT");
+            string accesskey = Environment.GetEnvironmentVariable("DS3_ACCESS_KEY");
+            string secretkey = Environment.GetEnvironmentVariable("DS3_SECRET_KEY");
+            string proxy = Environment.GetEnvironmentVariable("http_proxy");
+
+            if (string.IsNullOrEmpty(endpoint) || string.IsNullOrEmpty(accesskey) || string.IsNullOrEmpty(secretkey))
+            {
+                string fatalerr = "Must set values for DS3_ENDPOINT, DS3_ACCESS_KEY, and DS3_SECRET_KEY to continue";
+                Console.WriteLine(fatalerr);
+                if (clientSwitch.TraceWarning) { Trace.WriteLine(fatalerr); }
+                return;
+            } // if connection paramaters defined
+
+            // out-of-box examples in project
+            string bucket = "sdkexamples";
+            string testRestoreDirectory = "DataFromBucket";
+            string testSourceDirectory = "TestData";
+            string testSourceSubDirectory = "FScottKey";
+            string testSourceFile = "LoremIpsum.txt";
+            string binaryfile = "binarytest";
```

Comment:
variable names with more than one word should have capital letter of the second word :)
binaryfile ==> binaryFile